cypress/integration/tests/controls/jobfunctions/create.test.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference types="cypress" />
+
+import { login } from '../../../../utils/utils';
+import {Jobfunctions} from '../../../models/jobfunctions'
+
+describe('Create New Job Function', () => {
+    const jobfunctions = new Jobfunctions();
+
+    beforeEach(() => {
+        login();
+    });
+
+    it('jobfunctions crud', function() {
```

Comment:
If we are going to keep the CRUD operations in one file, the file name is not suitable (create.test.ts).
Can we decide on this?